src/shared/functions/index.ts

```diff
@@ -130,3 +130,30 @@ export function mapAuthorToSummary(author: Partial<User>): AuthorSummary {
     organization: author.organization,
   };
 }
+
+/**
+ * Title cases string
```

Comment:
This JSDoc description says nothing more than the function name does. Small nitpick here, but maybe something like: 
```suggestion
 * Capitalizes the first letter of each word in the provided string.
```